src/style.css

```diff
@@ -0,0 +1,126 @@
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Regular.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Bold.ttf);
+  font-weight: 600;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 30px 14px 15px;
```

Comment:
center content in your product card with flex properties, not margins

Reply:
added flex properties and fix-width=170px for classes "product-card__header" and "product-card__content".